lxd/instance/drivers/driver_lxc.go

```diff
@@ -8246,15 +8246,12 @@ func (d *lxc) Info() instance.Info {
 
 // Metrics returns the metric set for the LXC driver. It collects various metrics related to memory, CPU, disk, filesystem, and network usage.
 func (d *lxc) Metrics(hostInterfaces []net.Interface) (*metrics.MetricSet, error) {
-	state := instance.PowerStateStopped
 	isRunning := d.IsRunning()
-
-	if isRunning {
-		state = instance.PowerStateRunning
+	out := metrics.NewMetricSet(map[string]string{"project": d.project.Name, "name": d.name, "type": instancetype.Container.String()})
```

Comment:
Could this `metrics.NewMetricSet` be moved after the `if !isRunning`?

Reply:
Yes. fixed